packages/components/src/v-stack/hook.ts

```diff
@@ -8,14 +8,16 @@ import type { VStackProps } from './types';
 export function useVStack(
 	props: WordPressComponentProps< VStackProps, 'div' >
 ) {
-	const { expanded = false, ...otherProps } = useContextSystem(
-		props,
-		'VStack'
-	);
+	const {
+		expanded = false,
+		alignment = 'strech',
```

Comment:
Typo

```suggestion
		alignment = 'stretch',
```

Changing the default value of the `alignment` property to `stretch` is a breaking change, since prior to this PR the default value is `edge` (due to the `useHStack` hook) — can we roll this change back, and pass the `alignment = "stretch"` to `VStack` where necessary?

Reply:
I don't remember exactly the reasoning here but I believe the previous default value was broken or something like that and that I've used "stretch" here because that's what we expect as a result (the closest to the previous state)

Reply:
Yeah, we have a slight problem here 🤔 The previous VStack defaulted to
`justify-content: space-between` and `align-items: normal` (== `stretch` in most cases). This combination of styles does not exist in the newly corrected vertical `alignment` values.

I think I agree that we'll cause the least amount of damage by defaulting to `stretch` here. The new defaults will be `justify-content: center` and `align-items: stretch`, but I believe the justify will not affect the vast majority of usages because very few people set explicit heights on their VStacks.

Let's just fix the type docs 👉 https://github.com/WordPress/gutenberg/commit/d88a8c833e7e01cb6ad11a8b3346fe873a4c8afa